src/sentry/static/sentry/app/views/organizationDiscover/discover.jsx

```diff
@@ -212,14 +213,8 @@ export default class OrganizationDiscover extends React.Component {
     }));
 
     return (
-      <Discover className="organization-home">
-        <Header
-          p={2}
-          justify="space-between"
-          align="center"
-          className="sub-header"
-          style={{marginBottom: 16}}
-        >
+      <Discover>
+        <Header px={space(4)} py={2} className="sub-header">
```

Comment:
Since we decided to use our space system instead of the emotion one, can we do `py={space(2)}` here for consistency